test/providers/google.test.ts

```diff
@@ -65,4 +65,35 @@ describe('google', () => {
     expect(resolvedStyles).toMatchObject(styles)
     expect(resolvedWeights).toMatchObject(weights)
   })
+
+  it('respects glyphs option and resolves optimized font', async () => {
+    const unifont = await createUnifont([providers.google()])
+
+    const { fonts } = await unifont.resolveFont('Poppins', {
+      glyphs: ['Hello', 'World'],
+      styles: ['normal'],
+      weights: ['400'],
+      subsets: [],
+    })
+
+    // Do not use sanitizeFontSource here, as we must test the url parameters
+    expect(fonts).toMatchInlineSnapshot(`
```

Comment:
Since the URL parameters were the only way to verify that the correctly optimized fonts were resolved, I did a snapshot test.